Automattic/wp-calypso
# RTS: Use screenshots for previews

Requires D153316-code

## Proposed Changes

Simplifies the Readymade Templates prototype by using the new screenshots added by D153316-code.

## Why are these changes being made?

Because we can to build the most basic prototype possible, and rendering the patterns had some additional challenges.

## Testing Instructions

- Apply D153316-code to your sandbox
- Sandbox the API and `s0.wp.com`
- Use the Calypso live link below
- Go to `/patterns`
- Make sure the readymade templates are previewed with the new screenshots
- Make sure the look good and that the layout has not changed

## Pre-merge Checklist
